example/gardener-local/kind/cluster/values.yaml

```diff
@@ -1,5 +1,5 @@
 environment: skaffold # skaffold|local
-image: kindest/node:v1.24.7
+image: kindest/node:v1.25.3
```

Comment:
If you want to go down this path, you would need to setup e2e tests for all the seed versions we support. I don't see much benefits. Eventually, 1.27 will be supported, and then we'll probably update straight to 1.27 anyways. We just couldn't do so with 1.26 because of HVPA, otherwise we would have done this as well.